move/axelar_gateway/sources/gateway.move

```diff
@@ -542,6 +567,7 @@ fun test_setup() {
         operator: operator_result,
         messages,
         signers,
+        version_control: _,
```

Comment:
can you create another PR with test coverage for gateway upgradability?

Reply:
will do when everything is done